src/csf/transformCsf.ts

```diff
@@ -118,7 +131,14 @@ export const transformCsf = (
   }
   if (!clearBody) result = `${result}${code}\n`;
   if (allTests.length) {
-    const describe = makeDescribe(csf.meta.title, allTests);
+    if (beforeEachPrefixer) {
```

Comment:
why is this called both here and on L140?

Reply:
unneeded. removed!